landing-page/content/common/spark-quickstart.md

```diff
@@ -0,0 +1,370 @@
+---
+url: spark-quickstart
+toc: true
+aliases:
+    - "spark-quickstart"
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Getting Started With Spark and Iceberg
+
+Spark is currently the most feature-rich compute engine for Iceberg operations. 
+We recommend you to get started with Spark to understand Iceberg concepts and features with examples.
+You can also view documentations of using Iceberg with other compute engine under the
+[**Compute Frameworks**](/compute-frameworks) tab.
+
+## Docker Quickstart (Spark)
+
+The fastest way to get started is to use a docker-compose file to launch a local Spark cluster with a configured
+Iceberg catalog. Save the following into a file named `docker-compose.yml`.
+
+```yaml
+version: "3"
+
+services:
+  spark-iceberg:
+    image: tabulario/spark-iceberg
```

Comment:
I found https://hub.docker.com/r/tabulario/spark-iceberg
Maybe it is worth referencing it in the document

On a side note, can you please open source the `Dockerfile` for it? I don't seem to find it here https://github.com/tabular-io

Reply:
I updated this with the link to docker hub (we still have a pending item to add the README from the repo there). As for the docker file, it's in the spark directory of the repo: https://github.com/tabular-io/docker-spark-iceberg/blob/main/spark/Dockerfile